spec/migrations/20200211181329_drop_red_hat_domain_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_migration
+
+describe DropRedHatDomain do
+  let(:miq_ae_namespace_stub) { migration_stub(:MiqAeNamespace) }
+
+  migration_context :up do
+    it "drops domain, classes, " do
+      miq_ae_namespace_stub.create!(:name => 'RedHat', :parent_id => nil)
```

Comment:
If you add more things under here like child namespaces (and classes, etc), you can see they won't be removed.